packages/uniforms-bridge-zod/src/ZodBridge.ts

```diff
@@ -29,7 +29,10 @@ function isNativeEnumValue(value: unknown) {
 }
 
 export default class ZodBridge<T extends ZodRawShape> extends Bridge {
-  constructor(public schema: ZodObject<T>) {
+  constructor(
+    public schema: ZodObject<T>,
+    private provideDefaultLabelFromFieldName: boolean = false,
```

Comment:
Same here. But maybe it'd make sense to have the schema as the first argument and the options object as the second? I'm not sure.